### ISSUE TITLE
✍️ Write 

### ISSUE THREAD

**Issue by tatianamac** [tatianamac/selfdefined#82]

**Comment by tatianamac** [comment]
Use #70 for context.

- [ ] Share common resources
- [ ] Itemise different definition components (Avoid/Use Instead, etc) more fully
- [ ] Give do's and don't's

**Comment by ovlb** [comment]
Shall this also include a style guide? I noticed that the quotes used aren’t uniform at all. I wrote a toll once that might be able to help us style them during render without having to nitpick in PRs: [Typographizer JS](https://github.com/ovlb/typographizer-js) (Haven’t really tested this in production anywhere, though 🙈)

**Comment by tatianamac** [comment]
Yes, absolutely would include how to properly cite/quote (which will involve a "twin" frontend styling convention, which I love the idea of using Typographizer JS 🤓). Added as bullet point above.

**Comment by siawayforward** [comment]
Interested in the research/documentation aspect of this and would love to pair with a coder for thinking through how the information would appear. Already put out chirps in Slack. Also, point of clarification please - is the following statement of understanding correct?
This is unlike the [definition template](https://www.selfdefined.app/documentation/examples/flagged-word/) for how to define a word/phrase, but rather a resource for what to consider while addressing the fields in defining a word/phrase for contributors.

Otherwise looking forward to this finally! Thanks again for making this so inclusive @tatianamac  :)
